### PR TITLE
O3-4063: Ward App - update encounter types to use for Inpatient Notes…

### PR DESCRIPTION
**Pull request by mogoodrich**
… and Transfer Request

## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

File: packages/esm-ward-app/src/ward-workspace/ward-patient-notes/history/notes-container.component.tsx [hunk 1]
```diff
@@ -14,12 +14,12 @@ interface PatientNotesHistoryProps {
 
 const PatientNotesHistory: React.FC<PatientNotesHistoryProps> = ({ patientUuid, visitUuid }) => {
   const { t } = useTranslation();
-  const { emrConfiguration, isLoadingEmrConfiguration, errorFetchingEmrConfiguration } = useEmrConfiguration();
```
Comment (mogoodrich) [inline packages/esm-ward-app/src/ward-workspace/ward-patient-notes/history/notes-container.component.tsx:17]: Just a random unused variable picked up by the linter

File: packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-request-form.component.tsx [hunk 2]
```diff
@@ -65,7 +65,6 @@ export default function PatientTransferForm({
     formState: { errors, isDirty },
     control,
     handleSubmit,
-    getValues,
```
Comment (mogoodrich) [inline packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-request-form.component.tsx:68]: Random unused variable picked up by the linter.

**Review by chibongho — APPROVED** [review]
LGTM!

**Review by mseaton — APPROVED** [review]

**Comment by mogoodrich** [comment]
thanks @chibongho @mseaton ... the E2E tests are failing but this *seems* to be tangential, going to merge and see what happens...

**Pull request merged**